{
  "bug_location": "Unknown",
  "severity": 3,
  "categories": [
    "Unknown"
  ],
  "root_cause": "N/A - Analysis failed",
  "issue_number": 5411,
  "title": "Test test_server_pd_transfer_leader failed"
}